### ISSUE TITLE
Websocket connection closed after 60 secs

### ISSUE THREAD

**Issue by tobdub** [vernemq/vernemq#1292]
### Environment
- VerneMQ Version: 1.9.1
- OS: erlio/docker-vernemq:1.9.1-alpine
- VerneMQ configuration (vernemq.conf):
```
allow_anonymous = off
persistent_client_expiration = 1w
max_online_messages = 10000
max_offline_messages = 1000
listener.max_connections = 10000
plugins.vmq_webhooks = on
plugins.vmq_webhooks.priority = 1
vmq_webhooks.webhook_reg.hook = auth_on_register
vmq_webhooks.webhook_reg.endpoint = http://auth:5000
vmq_webhooks.webhook_sub.hook = auth_on_subscribe
vmq_webhooks.webhook_sub.endpoint = http://auth:5000
vmq_webhooks.webhook_pub.hook = auth_on_publish
vmq_webhooks.webhook_pub.endpoint = http://auth:5000
vmq_webhooks.webhook_pub.no_payload = on
plugins.vmq_diversity = on
vmq_diversity.auth_postgres.enabled = on
plugins.vmq_passwd = off
plugins.vmq_acl = off
plugins.vmq_bridge = off
log.console = console
log.console.level = debug
nodename = VerneMQ@127.0.0.1
distributed_cookie = vmq
```
- Cluster size/standalone: 1 node
- [paho js mqtt client 1.1.0](https://www.npmjs.com/package/paho-mqtt)
- Chrome 76, Firefox 68 running on Ubuntu 18.04

### Expected behavior
The websocket connection should not close when subscription messages are still being received.

### Actual behaviour
The websocket connection closes after 60 secs unless a message is published, even though messages are received and client keep alive is set to 5 min.
The problem only affects websocket connections, not a direct mqtt connection.
The problem happens both when connecting to the broker through a loadbalancer over the Internet and when running the broker locally over localhost.
The problem does not happen when connecting to a mosquitto broker over websockets using the same client.
The VerneMQ log just shows: ``14:05:47.091 [debug] stop due to disconnect``

**Comment by larshesel** [comment]
Hi, thanks for reporting. I was able to confirm the behaviour and we'll see if we can figure out what's going on. I suspect our recent upgrade of the websocket library might have something to do with it.

**Comment by larshesel** [comment]
Hi @tobdub we just merged a fix for this which will be part of VerneMQ 1.9.2. Thanks again for reporting!
